homeassistant/components/adax/climate.py

```diff
@@ -164,3 +180,12 @@ async def async_update(self) -> None:
         self._attr_target_temperature = data["target_temperature"]
         self._attr_current_temperature = data["current_temperature"]
         self._attr_available = self._attr_current_temperature is not None
+        if data["target_temperature"] == 0:
```

Comment:
Should we really allow to turn the heater off by setting target temp to 0?
It's not really logical + we can already turn off by `turn_off` or by `set_hvac_mode`

Reply:
This is inside the async update, so testing for target temp equal to 0 is testing if the heater is turned off. That is how the heater API reports that the heater is off, by reporting target temp equal to 0.

If you look inside set_hvac_mode, the actual way it is turned off is by setting target temp = 0 (line 167)

Reply:
Yes, I looked incorrectly. Thanks